lib/redux/actions/async.js

```diff
@@ -477,9 +483,22 @@ export function clickDeviceSelectionDone() {
           } else {
             let newProfile = actionUtils.mergeProfileUpdates(allUsers[uploadTargetUser], updates);
             dispatch(syncActions.updateProfileSuccess(newProfile, uploadTargetUser));
+            if (isClinicAccount) {
+              _.forEach(targetDevices[uploadTargetUser], function(device){
```

Comment:
This dispatches a metric for each target device even if it isn't _newly_ added, if I'm reading this right...? That seems like it will overreport the metric. I see the complication here with how the action works, so maybe the functionality/limitations should be discussed with BA?